WSL/basic-commands.md

```diff
@@ -168,8 +168,44 @@ To terminate the specified distribution, or stop it from running, replace `<Dist
 
 ## Identify IP address
 
-- `wsl hostname -i` for the IP address of your Linux distribution installed via WSL 2 (the WSL 2 VM address)
-- `cat /etc/resolv.conf` for the IP address of the Windows machine as seen from WSL 2 (the WSL 2 VM)
```

Comment:
We should use this command instead:
`ip route show | grep -i default | awk '{ print $3}'`

see here: https://learn.microsoft.com/en-us/windows/wsl/networking#ipv6-access